java/spring-framework/pom.xml

```diff
@@ -0,0 +1,40 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>benchmark.spring-framework</groupId>
+    <artifactId>benchmark</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>benchmark</name>
+    <description>Demo project for Spring Framework</description>
+    <packaging>war</packaging>
+
+    <parent>
+        <groupId>org.springframework</groupId>
+        <artifactId>spring-framework-bom</artifactId>
+        <version>5.2.0.RELEASE</version>
+    </parent>
+    <properties>
+        <java.version>1.8</java.version>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <failOnMissingWebXml>false</failOnMissingWebXml>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-webmvc</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+            <version>2.10.1</version>
+        </dependency>
+        <dependency>
+            <groupId>javax.servlet</groupId>
+            <artifactId>javax.servlet-api</artifactId>
+            <version>4.0.1</version>
```

Comment:
could you remove this ? 
it's add more maintenance (e.g : we do not care about which version of each dependency, but we care about version of **frameworks** in use)

Reply:
This is necessary as we are writing a servlet we must specify which version of the spec it is in.

Reply:
in fact I mean specify the version of dependency, not dependency itself

Reply:
In this case we should not update the dependency: The current spring version targets servlet 4.0 spec, which is the last one. Since spring 5.2 will not target any greater servlet version we can be sure this one does not need maintenance.

Reply:
fair enough

however, is there any way to say `use the latest version available` ?

Reply:
@david-m-s could you use `[4,5)` for  `javax.servlet` ?